nano/node/json_handler.cpp

```diff
@@ -3040,7 +3042,9 @@ void nano::json_handler::pending ()
 			{
 				auto mid = hash_amount_pairs.size () <= (offset + count) ? hash_amount_pairs.end () : hash_amount_pairs.begin () + offset + count;
 				std::partial_sort (hash_amount_pairs.begin (), mid, hash_amount_pairs.end (), [] (auto const & lhs, auto const & rhs) {
-					return lhs.second > rhs.second;
+					auto amount_l = lhs.second;
```

Comment:
This change seems to sort the blocks first by amount and then by hash.
However, the documentation does not mention anything about sorting by hash.
And I do not see much value in sorting by hash, we are just making our life more difficult for no obvious value gain.
So, I think we should not make this change unless it fixes something else that I have missed.